### PR TITLE
OpenCost as a FinOps Certified Solution and other news

### PR DESCRIPTION
**Pull request by mattray**

### CODE REVIEW

**Comment by mattray** [comment]
Waiting on additional content from @dwbrown2

**Comment by dwbrown2** [comment]
Quote added to draft doc!

**Comment by mattray** [comment]
@dwbrown2 quote added to PR, ready to publish when you are

File: blog/2023-02-27-finops-certified-solution/index.mdx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+slug: finops-certified-solution
+title: OpenCost is a FinOps Certified Solution!
+authors: [mattray]
+tags: [opencost, finops, cncf, scale]
+---
+
+It’s been a busy start to the year. We’re excited to announce that OpenCost has joined Kubecost as a [FinOps Certified Solution](https://www.finops.org/certifications/finops-certified-solution/). The [FinOps Foundation](https://www.finops.org) has certified OpenCost because it delivers Kubernetes cloud cost monitoring, a specific part of cloud cost management. According to Vas Markanastasakis, Principal at the FinOps Foundation: "Our focus at the FinOps Foundation is helping advance the people and the practice of cloud financial management. OpenCost's vendor-neutral open source project shares that focus."  We look forward to helping more practitioners in the community and working with the FinOps Foundation to drive open standards.
```
Comment (dwbrown2) [inline blog/2023-02-27-finops-certified-solution/index.mdx:8]: Is there a reason to keep the Kubecost reference? I think we should remove.

**Pull request merged**